trade-finance-manager-ui/server/controllers/deals/index.js

```diff
@@ -48,8 +49,7 @@ const queryDeals = async (req, res) => {
   let activeSortByField = '';
   let searchString = '';
   const { userToken } = req.session;
-
-  delete req.body._csrf;
+  const sortButtonWasClicked = (req.body?.formId === 'deals-table-sorting') ? true : false;
```

Comment:
const sortButtonWasClicked = req.body?.formId

Reply:
I simplified line 52.

Resolving this comment as PR was approved 1 hour ago.